server/src/services/Analysis.ts

```diff
@@ -1,29 +1,45 @@
+import { type } from 'os';
+
 class Analysis {
   average: number;
   median: number;
   standardDeviation: number;
 
-  constructor(array: number[]) {
-    this.average = array.reduce((a, b) => a + b, 0) / array.length;
-    this.median = this.countMedian(array);
-    this.standardDeviation = this.countStandardDeviation(array);
+  constructor(array: string[]) {
```

Comment:
może nazwiemy to voteValues?? generalnie to nazywanie zmiennej array ma mało sensu, zwłaszcza w ts gdzie obok masz typ podany.

Reply:
no w sumie to racja:)